package.json

```diff
@@ -15,6 +15,7 @@
   },
   "homepage": "https://thelounge.github.io/",
   "scripts": {
+    "coverage": "istanbul cover _mocha -r test/fixtures/env.js test/**/*.js",
```

Comment:
Would it work like `istanbul cover npm run test:mocha` here? I'd rather avoid duplication.